### DISCUSSION TITLE
RSS Guard forgets its maximized state

### DISCUSSION THREAD

**Discussion by slalaurette** [martinrotter/rssguard#1015] [General]
Every time I click on an article, RSS Guard gets unmaximized. Judging from the length of the scrollbar indicator, the window gets really really tall. Maximizing it again only has effect until I click on another article. (Clicking on the same, selected, article doesn't change the window.)

I  just added a long list of labels. I suspect this has something to do with this bug.

**Comment by martinrotter** [comment]
Do you use latest version of RSS Guard? This could be associated with long list of labels but I tried to reproduce and the bug does not happen here.

**Comment by slalaurette** [comment]
Hi, sorry for the delay! I was certain I was using the latest version but alas, I was not. The bug has been solved. Sorry for not checking, and thanks!
